nasa/ow_simulator
# Oceanwater 943 Update Stow and Unstow

## Linked Issues:
| EPIC ⚡ | [OCEANWATER-933](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-933) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-943](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-943) |
| Github :octocat:  | #302 |


### Summary of Changes
* Remove goal, feedback, and result components of Stow and Unstow actions.
* Rename these actions to ArmStow and ArmUnstow.
* Move these action files to owl_msgs package.

### Test
* This branch must be tested with the corresponding branch in `ow_autonomy`.
* Do a clean build.  This itself verifies most of renaming and moving.
* Start any simulator world
* Run `arm_unstow_action_client.py` and see that the arm unstows normally.
* Run `arm_stow_action_client.py` and see that the arm stows normally.
